bundle/Dockerfile

```diff
@@ -0,0 +1,239 @@
+# For versions pinning, use the ./version.sh script
```

Comment:
I Would like to see the dev images as well ( development + test )

Reply:
Noted! I will add them. I will make a PR on dev/0.27 to change the generator in order to avoid overrides and use the generated app. This will help to build docker images in a sustainable way.

Reply:
@alecslupu I've created an update script to generate Docker images, docker-compose files and update the readme. This way we can have several branches (dev, 0.27-stable, 0.26-stable) for now. 

Regarding images for development/testing purpose, I still need to work on it.